### DISCUSSION TITLE
port 465

### DISCUSSION THREAD

**Discussion by samrudd** [postalserver/postal#1757] [Feature suggestions]
it would be great if we could use port 465 to speak to the mail servers on the postal service. whmcs sits with an errro for not being secure.

**Comment by willpower232** [comment]
TLS is already available on port 25, changing the port number wouldn't magically make it more secure.

Have you configured `tls_enabled`?
https://github.com/postalserver/postal/blob/ea10ae18f10ec243c9e30a540a3de4d853155838/config/postal.defaults.yml#L61-L66

If you have, what error message are you seeing in whmcs?

**Reply by samrudd** [reply]
So do I just set this to true and restart postal? I neglected to see that option. Whmcs just says it’s not secure

**Reply by willpower232** [reply]
you have to supply a certificate file as well but essentially yes

**Reply by netstx** [reply]
@willpower232 is it possible to use port 25 for non-TLS, and 465 for TLS, concurrently?

**Comment by kluvi** [comment]
You are mixing 2 different things
1) STARTTLS
- = encrypted communication inside unencrypted SMTP protocol
- this is supported by Postal - see @willpower232 answer
- runs always on same port as unencrypted SMTP (port 25 by-default)

2) SSL/TLS
- = unencrypted communication (SMTP protocol) inside encrypted SSL/TLS tunnel
- this must run on another port, than default 25
- you can use "hack" described here: https://github.com/postalserver/postal/discussions/2132 (there are 2 solutions - directly on host, or using another container)

**Reply by netstx** [reply]
@kluvi than you for the clarification :)

**Reply by ThatProgrammerr** [reply]
Lifesaver.
